standard/expressions.md

```diff
@@ -1480,6 +1487,34 @@ parenthesized_expression
 
 A *parenthesized_expression* is evaluated by evaluating the *expression* within the parentheses. If the *expression* within the parentheses denotes a namespace or type, a compile-time error occurs. Otherwise, the result of the *parenthesized_expression* is the result of the evaluation of the contained *expression*.
 
+### §tuple-expressions-new-clause Tuple expressions
```

Comment:
Tuples depend on `ValueTuple` with all the arities.  That should be added to the Annex on standard types.

Reply:
I've added them all in the upcoming commit. There is still no explanation for how longer tuples are chained.